### ISSUE TITLE
How to add custom butttom

### ISSUE THREAD

**Issue by AbdurRahamanAR** [alexandrtovmach/react-twitter-login#2]

**Comment by alexandrtovmach** [comment]
@AbdurRahamanAR Currently is there no way. I'll mark it as a feature request and implement in a few days, or if you'd like to make PR I'm ready to review it

**Comment by AbdurRahamanAR** [comment]
I am a junior developer. But i will try

**Comment by alexandrtovmach** [comment]
@AbdurRahamanAR Thank you for effort, but I'm already implemented =) Just had a few free minutes

It's available in [v1.1.1](https://www.npmjs.com/package/react-twitter-login/v/1.1.1):
1. Update package `npm i react-twitter-login@latest`
2. Use it:
```jsx
<TwitterLogin>
{/* code of your custom button */}
</TwitterLogin>
```

**Comment by AbdurRahamanAR** [comment]
Thank's a lot
